x-pack/plugin/src/test/resources/rest-api-spec/test/indices.freeze/10_basic.yml

```diff
@@ -0,0 +1,128 @@
+---
+"Basic":
+
+- skip:
+    version: " - 6.99.99"
+    reason: types are required in requests before 7.0.0
```

Comment:
Can we change the reason here to mention Freeze/Unfreeze API?

Reply:
well that is the reason. We will have the same test in 6.x but with types and a different reason?

Reply:
Ok